macros/edr/data_monitoring/data_monitors_configuration/get_column_monitors.sql

```diff
@@ -16,8 +16,7 @@
     {%- set columns_from_relation = adapter.get_columns_in_relation(model_relation) -%}
```

Comment:
There is a bug in the macro above (in get_column_monitors) where it always provides None to the column_tests